driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -619,6 +599,12 @@ public String toString() {
         return "CCM cluster " + clusterName;
     }
 
+    @Override
+    protected void finalize() throws Throwable {
```

Comment:
Just in case we miss a CCMBridge instance that isn't stopped when garbage-collected (shoudn't happen)